Code/src/main/java/nl/utwente/group10/ui/components/blocks/DisplayBlock.java

```diff
@@ -144,7 +144,12 @@ public int getInputIndex(InputAnchor anchor) {
         return 0;
     }
 
-    public void error() {
-        this.getStyleClass().add("error");
+    @Override
+    public void setError(boolean error) {
+        if (error) {
+            this.getStyleClass().add("error");
+        } else {
+            this.getStyleClass().removeAll("error");
+        }
```

Comment:
Deze code wordt gedupliceerd in elk `Block` lijkt het - misschien een idee om 'm niet abstract te maken?